doc/60_Dashboard.md

```diff
@@ -4,7 +4,7 @@ Gitpod's dashboard allows you to log in and out, navigate through all your works
 details of your account and subscription. It can be reached by pointing your browser to
 [https://gitpod.io](https://gitpod.io).
 
-If you're not logged in, you will see a _Log in_ button. Otherwise, the top-right corner will
+If you are not logged in, you will see a _Log in_ button. Otherwise, the top-right corner will
```

Comment:
Did you do a find/replace for all occurrences of _don't_, _can't_, _doesn't_, _isn't_, _it's_ etc?

Reply:
yes, in the follow-up :-)